src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+import { TodosList } from '../../types/TodoList';
+
+type Props = {
+  todo: TodosList,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const found = (userId:number) => {
+    return usersFromServer.find(user => user.id === userId)
+    || usersFromServer[0];
+  };
```

Comment:
create **new array** of todos with `users` in App.tsx and pass it to `TodoList` to not do it here
```suggestion
```